src/tools/wasm-opt.cpp

```diff
@@ -293,13 +303,18 @@ int main(int argc, const char* argv[]) {
     outfile << generateJSWrapper(wasm);
     outfile.close();
   }
-
   if (emitSpecWrapper.size() > 0) {
     std::ofstream outfile;
     outfile.open(emitSpecWrapper, std::ofstream::out);
     outfile << generateSpecWrapper(wasm);
     outfile.close();
   }
+  if (emitWasm2CWrapper.size() > 0) {
+    std::ofstream outfile;
+    outfile.open(emitWasm2CWrapper, std::ofstream::out);
+    outfile << generateWasm2CWrapper(wasm);
+    outfile.close();
```

Comment:
It could be a new tool I suppose, but we have 3 wrapper generators now (spec, js, wasm2c), and it's convenient to run them from wasm-opt so you can emit the wrapper as you generate the fuzz code, in a single invocation.

Reply:
idk, it seems similar to a lot of other auxiliary functionality in wasm-opt like the fuzzing stuff or the JS wrappers.

Reply:
Oh I see, there is a precedent here.  In that case that fine for now.

But maybe as a followup this should be a separate tool?    So you would `wasm-opt` and then `wasm-generate-fuzz-wrapper -type=wasm2c` or something less clunky than that.

Reply:
Yeah, maybe that's better. Another option would be to make all 3 of those be passes.